.github/workflows/rust.yml

```diff
@@ -0,0 +1,80 @@
+name: Rust
+on: [pull_request]
+jobs:
+  cleanup-runs:
```

Comment:
there is no way to stop ci when a new commit is pushed currently built into GitHub, this is meant as a workaround for now.